crates/forge/tests/integration/test_state.rs

```diff
@@ -126,10 +136,215 @@ fn test_simple_syscalls() {
             "WarpChecker".to_string(),
             Path::new("tests/data/contracts/warp_checker.cairo"),
         )
-        .unwrap()
+        .unwrap(),
+        Contract::new(
+            "SequencerAddressChecker",
+            indoc!(
+                r#"
+                #[starknet::contract]
+                mod SequencerAddressChecker {
+                    use result::ResultTrait;
+                    use starknet::{ ClassHash, library_call_syscall, ContractAddress};
+
+                    #[storage]
+                    struct Storage {
+                    }
+
+                    #[external(v0)]
+                    fn get_sequencer_address(ref self: ContractState) -> ContractAddress {
+                        starknet::get_block_info().unbox().sequencer_address
```

Comment:
Isn't this handled on the cheatnet side? I wonder if it makes more sense to have a test for that there.

Reply:
? It tests consistency of values between cheatnet and tests